static/docs/user-guide/autocomplete.md

```diff
@@ -73,6 +73,36 @@ fi
 You can `source` your `~/.bash_profile` or launch a new terminal to utilize
 completion.
 
+<details>
+
+### Click to expand if it doesn't work on Debian/Ubuntu
+
+As mentioned above, it should work out of the box. But if it does not, try these steps:
+
+- Make sure that the package `bash-completion` is installed:
+  ```bash
+  sudo apt install --reinstall bash-completion 
+  ```
+- Make sure that it is enabled. Edit `~/.bashrc` and make sure that these lines are there:
+  ```bash
```

Comment:
here `bash` or `sh` make sense

Reply:
or `shell` .. I'm not sure if the difference is important, check some other places in the docs. We should use the one the highlighter is on for.